lib/sparql_to_sw_solr/instance_topic_fields.rb

```diff
@@ -0,0 +1,38 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module InstanceTopicFields
+
+      private
+
+      def add_doc_topic_fields(doc)
+        doc[:topic_search] = topics
```

Comment:
actually, topic_search should be the entire value of the string:  `Conductors (Music)--Italy--Biography.`  The splitting is only for `topic_facet` if you re-read the googledoc spec.  You can also see that they are different by comparing https://github.com/sul-dlss/solrmarc-sw/blob/master/stanford-sw/sw_index.properties#72 with https://github.com/sul-dlss/solrmarc-sw/blob/master/stanford-sw/sw_index.properties#L87.  There are a lot of subfields in marc for these, so "allAlphaExcept" is basically a-u, but the topic facet is only abcdq ...